ADDONS.md

```diff
@@ -46,13 +46,11 @@ Foreground and background colors do not replace one another but they sum up, whi
 
 So the following sequence, which is not terminated by reset, `\u001B[41m``AAA``\u001B[40m``\u001B[37m``BBB` produces as output:
 
-<span style="color:white;background-color:black; display:inline-block; width:100%"><span style="color:black;background-color:red;display:inline-block">AAA</span>BBB    
-</span>
+![withoutreset](colored-message-without-reseting.png "")
```

Comment:
I like the idea of using images, but this output is not 100% correct. The '`' characters belong to the Markdown format and not the output. The correct image is:
<img width="862" alt="Screen Shot 2020-06-25 at 20 38 52" src="https://user-images.githubusercontent.com/1557512/85779731-31c63380-b724-11ea-91bb-08d6bc23fa9a.png">